metrics/mypy_high_water_mark

```diff
@@ -1 +1 @@
-91.9000
\ No newline at end of file
+91.8600
```

Comment:
Some of the validation code, ironically, isn't typable by mypy. I propose to bump this rather than further clutter up the PR (I'm running out of cheap tricks to ratchet mypy coverage, too...).